### DISCUSSION TITLE
having problems with creating superuser and connecting storefront

### DISCUSSION THREAD

**Discussion by Brightlight0721** [saleor/saleor#8855] [Deployment]
hello,

Q1)

I installed Saleor and logged in with admin@example.com / admin

and changed the email and password in the dashboard account setting is that how to make superuser?

I tried 'python manage.py createsuperuser' but it did not work.. when I put the code in cmd, it just returns 'python'..

how to make superuser?

Q2)

the browser can not connect to http://localhost:3000/ which is the storefornt..

what should I do?

please help me..

Thank you for your generosity

**Comment by Brightlight0721** [comment]
Dont worry guys I solved the problems!

**Reply by siraphobk** [reply]
I also faced the same Q2 problem. Would you please share your solution?

**Comment by Brightlight0721** [comment]
​​
Hi,

I found the video on youtube, and followed the steps

Hope you solve the problem!  

Video link:

https://youtu.be/jQUNaSD-3yU



​
-----Original Message-----
